### ISSUE TITLE
Lightweight WebConsole Web App

### ISSUE THREAD

**Issue by DerZerstampfer** [mesacarlos/WebConsole#55]
Hey i wanted to ask if you could make a lightweight version of the Web App,
i really like this Project but wanted to implement it into another Website.
With lightweight i mean a version where you input the ip, port and Password and then connect, without saving any data and a function like "startWebConsole" with these values: ip, port, password, consoleTextArea, commandInputForm, whereItShouldShowRam, whereItShouldShowCPU, whereItShouldShowPlayer

So basically a developer friendly version that i only need to send all values to the function andWebConsole takes care of everything else

**Comment by mesacarlos** [comment]
Yeah sure. The client code is already done with this purpose in mind. It is composed of this files:
scripts/
scripts/

**Comment by mesacarlos** [comment]
Yeah, it is already done with this purpose in mind. It is composed of this files:
**scripts/WebConsoleConnector.js** This module connects to the server. It is basically a WebSocket client.
**scripts/WebConsoleManager.js** This keeps all the opened connections in one place and is the "API" that the rest of the files use to communicate with the server.
**scripts/WebConsole.js** Part of the JavaScript code of the client is here. You don't probably need this.
**scripts/WebConsoleJqueryHandler.js** JQuery code is here. You don't probably need this.
**scripts/WebConsoleLanguage.js** This manages the client translations. You don't probably need this.
**scripts/WebConsolePersistenceManager.js** This saves the servers into the browser. You don't probably need this.

I just made a small "API" for you using the mentioned two necessary files.
[WebConsoleJS-API.zip](https://github.com/mesacarlos/WebConsole/files/5489519/WebConsoleJS-API.zip)
